babel.config.js

```diff
@@ -0,0 +1,35 @@
+const presets = [
+  [
+    '@babel/preset-env',
+    {
+      targets: {
+        browsers: [
+          '>0.25%',
+          'not ie 11',
+          'not op_mini all',
+          'safari >= 11',
+          'not safari 5.1'
```

Comment:
Let's discuss the support range. For sure, I would not support unsupported browsers `'no_dead'`.